cantabular-import/deps.yml

```diff
@@ -4,19 +4,26 @@ services:
     image: mongo:3.6
     ports:
       - 27017:27017
+    volumes:
+      - ./mongodb/entrypoint-initdb.d/init.js:/docker-entrypoint-initdb.d/init.js:ro
```

Comment:
for the benefit of the casual on-looker, can yo add a commend before this line to say exactly what its doing (and maybe why)